apps/feedback/models/feedback_form.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+/*
+ * TODO Add support for versioning
+ */
+type FeedbackForm struct {
+	gorm.Model
+	Title       string `gorm:"type:text; not null"`
+	Description string `gorm:"type:text;"`
+	Status      int8   `gorm:"default:0; not null"`
```

Comment:
Same as other Status field comment..

Reply:
Added todo for now, will tackle after API